contact.html

```diff
@@ -12,14 +12,18 @@
     <div class="main-container">
         <header class="header">
             <div class="logo">
-                <div class="name">EDAH ŠAHINOVIĆ</div>
+                <div class="name">
+                    <a href="/">EDAH ŠAHINOVIĆ</a>
```

Comment:
Made the logo/name/title be a link instead of just text. Usually logos link to the homepage.